### PR TITLE
Pay 4106.1

### PR DESCRIPTION
**Pull request by arbab-cog**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/src/main/java/uk/gov/hmcts/fees2/register/util/FeeFactory.java [hunk 1]
```diff
@@ -16,6 +17,8 @@ public static Fee getFee(FeeDto request){
             fee = new RateableFee();
         if (request instanceof RelationalFeeDto)
             fee = new RelationalFee();
+        if (request instanceof RelationalFeeDto || request instanceof LoaderRelationalFeeDto)
```
Comment (prashant-khunte) [inline api/src/main/java/uk/gov/hmcts/fees2/register/util/FeeFactory.java:20]: Duplicate instance check

File: api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/RelationalFeeDto.java [hunk 2]
```diff
@@ -1,14 +1,76 @@
...
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import uk.gov.hmcts.fees2.register.api.contract.FeeVersionDto;
 
 @Data
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 public class RelationalFeeDto extends FixedFeeDto {
```
Comment (prashant-khunte) [inline api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/RelationalFeeDto.java:14]: verify RelationalFeeDto extends FixedFeeDto relationship exist

Comment (prashant-khunte) [inline api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/RelationalFeeDto.java:14]: verify RelationalFeeDto extends FixedFeeDto relationship exist

File: api/src/main/java/uk/gov/hmcts/fees2/register/util/FeeFactory.java (hunk 1 above)
Comment (arbab-cog) [reply api/src/main/java/uk/gov/hmcts/fees2/register/util/FeeFactory.java:18]: Removed the duplicate check

**Review by arbab-cog — COMMENTED** [review]
existing class

**Review by AnilDhawaleGitHub — APPROVED** [review]

**Review by prashant-khunte — APPROVED** [review]

**Review by AyanDuttaHMCTS — APPROVED** [review]

**Pull request merged**